extensions/cornerstone-dicom-rt/src/loadRTStruct.js

```diff
@@ -39,6 +39,7 @@ async function checkAndLoadContourData(instance, datasource) {
           StudyInstanceUID: instance.StudyInstanceUID,
           SeriesInstanceUID: instance.SeriesInstanceUID,
           SOPInstanceUID: instance.SOPInstanceUID,
+          clientName: instance?.clientName,
```

Comment:
Why do you need to specify the clientName here in the dicom rt extension?

Reply:
Both dicom rt and dicom seg perform late loadings in data, but in dicom rt case it didnt pass the instance. So I passed the clientName to indicate which client is required to download the additional information